modules/tinymce/src/plugins/pagebreak/main/ts/core/FilterContent.ts

```diff
@@ -5,24 +5,31 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import Editor from 'tinymce/core/api/Editor';
 import Env from 'tinymce/core/api/Env';
 import * as Settings from '../api/Settings';
 
-const getPageBreakClass = () => 'mce-pagebreak';
+const pageBreakClass = 'mce-pagebreak';
 
-const getPlaceholderHtml = () => {
-  return '<img src="' + Env.transparentSrc + '" class="' + getPageBreakClass() + '" data-mce-resize="false" data-mce-placeholder />';
+const getPlaceholderHtml = (shouldSplitBlock: boolean) => {
+  const html = `<img src="${Env.transparentSrc}" class="${pageBreakClass}" data-mce-resize="false" data-mce-placeholder />`;
+  if (shouldSplitBlock) {
+    return `<p>${html}</p>`;
+  } else {
+    return html;
+  }
```

Comment:
This could be a ternary maybe?
```suggestion
  return shouldSplitBlock ? `<p>${html}</p>` : html;
```